pkg/cliplugins/kubestellar/syncer-gen/edgesync.go

```diff
@@ -459,14 +468,14 @@ func (o *EdgeSyncOptions) enableSyncerForWorkspace(ctx context.Context, config *
 		},
 		Subjects: subjects,
 		RoleRef:  roleRef,
-	}, metav1.CreateOptions{}); err != nil && !apierrors.IsAlreadyExists(err) {
+	}, metav1.CreateOptions{FieldManager: fieldManager}); err != nil && !apierrors.IsAlreadyExists(err) {
 		return "", "", nil, err
 	}
 
 	// Wait for the service account to be updated with the name of the token secret
 	tokenSecretName := ""
 	var serviceAccount *corev1.ServiceAccount
-	_ = wait.PollImmediateWithContext(ctx, 100*time.Millisecond, 3*time.Second, func(ctx context.Context) (bool, error) {
+	_ = wait.PollImmediateWithContext(ctx, 300*time.Millisecond, o.ServiceAccountTokenWait, func(ctx context.Context) (bool, error) {
```

Comment:
This 0.3 second duration seems to be different from line 485 ("we wait for 3 seconds").

Reply:
Oh never mind, I see that the previous 3s timeout is now configurable by using `ServiceAccountTokenWait`.